frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -93,6 +100,46 @@ export const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
       optional: true,
     });
   }
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const fetchHelmReleases = async () => {
+      let releases: HelmRelease[];
+      try {
+        releases = await coFetchJSON(`/api/helm/releases?ns=${namespace}`);
+      } catch {
+        return;
+      }
+      if (ignore) return;
+
+      const releaseResourcesMap = releases.reduce((acc, release) => {
+        const manifestResources: K8sResourceKind[] = safeLoadAll(release.manifest);
+
+        manifestResources.forEach((resource) => {
+          const resourceKindName = `${resource.kind}---${resource.metadata.name}`;
```

Comment:
create a small utility for release key? It's being used mutiple times.
```
const createHelmReleaseKey = (resource) => `${resource.kind}---${resource.metadata.name}`;
```

Reply:
done